src/_pyinstaller_hooks_contrib/hooks/stdhooks/hook-clr.py

```diff
@@ -18,23 +18,34 @@
 
 
 import ctypes.util
-from PyInstaller.compat import is_win, getsitepackages
-from os.path import join, exists
+from PyInstaller.compat import is_win
+from PyInstaller.utils.hooks import logger
```

Comment:
Take the logger from `PyInstaller.log` please.
```suggestion
from PyInstaller.log import logger
```
It's not supposed to be available in `PyInstaller.utils.hooks` - rather some functions in there use the logger and there's no proper namespace hiding so it looks like public API when it isn't.